man/systemd.network.xml

```diff
@@ -2461,9 +2450,40 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
       </varlistentry>
       <varlistentry>
         <term><varname>BindToInterface=</varname></term>
-        <listitem>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
-        to its network interface and all socket communication will be restricted to this interface.
-        Defaults to <literal>yes</literal>.
+        <listitem>
+          <para>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
+          to its network interface and all socket communication will be restricted to this interface.
+          Defaults to <literal>no</literal>, if <varname>RelayTarget=</varname> is set(see below),
```

Comment:
Maybe it is better to drop comma before `if`.

Missing white space before `(`.